gf-webapp-admin/src/component/rooms/room/services/roomServices.ts

```diff
@@ -6,8 +6,8 @@ export const changeRoomServices = (
   selectedService: ApartmentServices,
 
 ) => {
-  return roomServices.indexOf(selectedService) === -1 ?
-    [...roomServices, selectedService]
+  return roomServices.includes(selectedService) ?
+    roomServices.filter(value => value !== selectedService)
     :
-    roomServices.filter(value => value !== selectedService);
+    [...roomServices, selectedService];
```

Comment:
* `A little strange formatting. Maybe it's will look better with if-else statement instead of ternarian operator? What do u think?`
ye, if-else looks more readeble.
* `naming: "value" is a bad naming` 
Done
* `code duplication (gf-webapp-admin/src/component/sideMenu/addRoomModal/services/roomServices.ts)` 
* i'd like to solve this in #43 issue